test/Benchmarks/src/Compiler/Compile.enso

```diff
@@ -0,0 +1,67 @@
+from Standard.Base import all
+
+from Standard.Test import Bench
+
+polyglot java import java.lang.System as Java_System
+polyglot java import java.io.File as Java_File
+
+type Data
+    Value ~enso_bin:File ~std_base:File
+
+    bench_compile_base self =
+        self.startup [ "--no-compile-dependencies", "--no-global-cache", "--compile", self.std_base.to_text]
```

Comment:
What about `--no-ir-caches`?

Reply:
Yes, but the default generator in sbt tries to be as fast as possible (so for example it does not disable IR caches). Whereas in this benchmark, you try to do a cold, reproducible, compilation.

Reply:
> The [default generator in sbt](https://github.com/enso-org/enso/blob/27a535f6d068f0a2be9f9825b7b5214e8f9ba5ff/project/DistributionPackage.scala#L240) isn't using them either. I want to be close to "user experience" - e.g. our developer experience.

The major motivation for this PR is 

> generator in sbt tries to be as fast as possible

to make sure the _generator in sbt is as fast as possible_. That's why this new benchmarks mimics what the _generator in sbt_ does as closely as possible.